kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/Listable.java

```diff
@@ -15,7 +15,10 @@
  */
 package io.fabric8.kubernetes.client.dsl;
 
+import io.fabric8.kubernetes.api.model.ListOptions;
+
 public interface Listable<T> {
   T list();
   T list(Integer limitVal, String continueVal);
```

Comment:
Since the new method already covers the functionality of this one, it maybe good to add a `@Deprecated` annotation (plus some JavaDoc indicating to use the new method) so we can remove it in the next major release.